data/pokemon/29.json

```diff
@@ -59,13 +59,13 @@
     "name": "generation-i"
   },
   "names": {
-    "ja": "ニドラン♀",
-    "en": "Nidoran♀",
-    "fr": "Nidoran♀",
-    "it": "Nidoran♀",
-    "de": "Nidoran♀",
-    "es": "Nidoran♀",
-    "ko": "니드런♀"
```

Comment:
I wonder if we should still keep these given that they're actually part of the species name.